doc/examples/config/single.yaml

```diff
@@ -1,12 +1,15 @@
 credentials:
   users:
     guest:
-      roles: [super]
+      roles: [sharding]
```

Comment:
It seems, these are forgotten experimental changes.

Reply:
True, fixed.